README.md

```diff
@@ -84,6 +84,14 @@ as ensuring that it's refreshed when needed. For the default instances of
 the `DescopeSessionManager` class this means using the `EncryptedSharedPreferences`
 for secure storage of the session and refreshing it a short while before it expires.
 
+**Important**: If you're planning on using the default session storage, make sure to provide
+the application context when initializing `Descope` with a project ID / config:
+
+```kotlin
+Descope.config = DescopeConfig(projectId = "<PROJECT-ID>", baseUrl = "https://my.custom.domain")
```

Comment:
Maybe remove the mention of `baseUrl` here since it's not related to passkeys? (and doesn't need to be used in normal circumstances)